### ISSUE TITLE
Filter blocking directory adv

### ISSUE THREAD

**Issue by inesbel** [easylist/easylist#925]
Hi,
I'm having difficulties in building a site with Wordpress in a subdir named ADV [http://pirunel.net/adv]
The browser Firefox with ADBlock active, since the first step of Worpress installation (when you choose the language) displays the site ignoring the stylesheet.
Follows a snapshot
![wordpress_pirunel](https://user-images.githubusercontent.com/36738726/36550061-94e4f5c2-17f4-11e8-812c-edd77aedfa8a.jpg)

Someone in the ADBlock forum told me that the issue could be solved fixing easylist...
Thanks
                 Ines

**Comment by ryanbr** [comment]
Cant reproduce it, which filters show up ?

**Comment by inesbel** [comment]
Sorry, I'm not able to answer you. I've changed the subdir name (from adv to www), because I had to work on that site.
I can only tell you that I encountered that issue in Firefox, where there was ABP Adblock Plus installed.
Other browsers (without Adblock) showed the site well.
I asked for help in ADBlock forum and someone told me that the problem was about easylist, suggesting me to signal it in GitHub.
I'm a self-made wabmaster, not so expert in coding, so I cannot tell you more than this.
